### PR TITLE
Remove GA feature gates in 1.28 about storage

### PR DESCRIPTION
**Pull request by wlq1212**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Remove GA feature gates in 1.28 about storage

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by bart0sh** [comment]
@wlq1212 please, provide release note

**Comment by wlq1212** [comment]
> @wlq1212 please, provide release note

Added

File: pkg/volume/csi/csi_plugin.go [hunk 1]
```diff
@@ -228,9 +229,6 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 		csitranslationplugins.AzureDiskInTreePluginName: func() bool {
 			return true
 		},
-		csitranslationplugins.AzureFileInTreePluginName: func() bool {
-			return utilfeature.DefaultFeatureGate.Enabled(features.CSIMigrationAzureFile)
```
Comment (jsafrane) [inline pkg/volume/csi/csi_plugin.go:232]: Don't delete this chunk, return `true` instead.

File: pkg/volume/csimigration/plugin_manager.go [hunk 2]
```diff
@@ -92,8 +93,6 @@ func (pm PluginManager) IsMigrationEnabledForPlugin(pluginName string) bool {
 		return true
 	case csilibplugins.GCEPDInTreePluginName:
 		return pm.featureGate.Enabled(features.CSIMigrationGCE)
-	case csilibplugins.AzureFileInTreePluginName:
-		return pm.featureGate.Enabled(features.CSIMigrationAzureFile)
```
Comment (jsafrane) [inline pkg/volume/csimigration/plugin_manager.go:96]: Again, return `true`.

**Comment by bart0sh** [comment]
@wlq1212 please [sign the CLA](https://github.com/kubernetes/community/blob/master/contributors/guide/README.md#sign-the-cla) and fix CI test failures, thanks.

**Comment by dims** [comment]
@wlq1212 please sign the CLA and rebase the PR to latest master branch.

**Pull request closed without merging**